docs/changelog.md

```diff
@@ -166,42 +166,42 @@
 ## 16.0.0
 
 - Breaking: the release files are not under the `/release` folder any more. Also, paths to `themes` and `assets` have changed. If you are importing any assets or styles from the release folder, you would need to update the paths.
-- Chore: Migrates the project to angular-cli, making it compatible with Ivy (#1738)
-- Feature: Implements global module config via forRoot (#1688)
-- Feature: Colors of material style can be modified by target application (#1651)
-- Bug: Fixes Exception in scoller component destroy (#1685)
-- Bug: Fixes issue 1686: column resizing issue in force columns mode (no horizontal scroll) when container get smaller (#1687)
-- Docs: Updates demo pages (#1738)
+- Chore: Migrates the project to angular-cli, making it compatible with Ivy (swimlane/ngx-datatable#1738)
+- Feature: Implements global module config via forRoot (swimlane/ngx-datatable#1688)
+- Feature: Colors of material style can be modified by target application (swimlane/ngx-datatable#1651)
+- Bug: Fixes Exception in scoller component destroy (swimlane/ngx-datatable#1685)
+- Bug: Fixes issue 1686: column resizing issue in force columns mode (no horizontal scroll) when container get smaller (swimlane/ngx-datatable#1687)
+- Docs: Updates demo pages (swimlane/ngx-datatable#1738)
 
 ## 15.0.2
 
-- Fix: Fixes issue where footer pagination does not work when virtual scrolling is enabled (#1704)
+- Fix: Fixes issue where footer pagination does not work when virtual scrolling is enabled (swimlane/ngx-datatable#1704)
 
 ## 15.0.1
 
-- Fix: Fixes issue where using conditional templates would not render them (#1701)
+- Fix: Fixes issue where using conditional templates would not render them (swimlane/ngx-datatable#1701)
 
 ## 15.0.0
 
-- Breaking: Upgrade to angular 8 (#1697)
-- Fix: Module not found: Error: Can't resolve '.' (#1646)
-- Fix: trackByProp can't work. (#1622)
+- Breaking: Upgrade to angular 8 (swimlane/ngx-datatable#1697)
+- Fix: Module not found: Error: Can't resolve '.' (swimlane/ngx-datatable#1646)
+- Fix: trackByProp can't work. (swimlane/ngx-datatable#1622)
 
 ## 14.0.0
 
 - Breaking: Upgrade to angular 7
-- Fix: Fix compile error when using TemplateTypeCheck set to true (#1520)
+- Fix: Fix compile error when using TemplateTypeCheck set to true (swimlane/ngx-datatable#1520)
 
 ## 13.1.0
 
-- Feature: Added support for tree grid (#1273)
-- Fix: Improve virtualization as a flag (#1297)
-- Fix: Fix issue when count is undefined #838 (#1499)
-- Fix: Fix usage withing prerendered SPA (#1438)
+- Feature: Added support for tree grid (swimlane/ngx-datatable#1273)
+- Fix: Improve virtualization as a flag (swimlane/ngx-datatable#1297)
+- Fix: Fix issue when count is undefined #838 (swimlane/ngx-datatable#1499)
```

Comment:
One link missed.

Reply:
I've dropped the ticket in favor of the PR which itself clearly covers this issue (and links to it).

Reply:
@jsoref I sorry, but I think we should not change the changelog. May I ask to keep and correct the lint to the issue?